driver/pace/driver/run.py

```diff
@@ -6,8 +6,7 @@
 import yaml
 
 from ndsl.logging import AVAILABLE_LOG_LEVELS, ndsl_log
-
-from .driver import Driver, DriverConfig
+from pace.driver.driver import Driver, DriverConfig
```

Comment:
What's the reason for this change?

Reply:
I am following the style that was set forth in how I handled imports in the submodules. When I am within a package and I am referencing something internal to the package, the absolute path is used, when outside the package the relative path is used.

Reply:
Yeah that makes sense, it just looks funny here